docs/about/community.md

```diff
@@ -10,19 +9,44 @@ custom_edit_url: null
 
+Join the conversation in the Open Source with SLU [Slack workspace](https://join.slack.com/t/oswslu/shared_invite/zt-24f0qhjbo-NkSfQ4LOg5wXxBdxP4vzfA).
+
+## Overview
+
+* Join our Slack workspace.
+* Contribute changes to an Open Source with SLU project.
+* Add your name to the Contributors list.
+* After making five contributions, you can be added to the `oss-slu` team in GitHub by contacting one of the program staff.
+
+## Getting Startted
+
+The success of our projects depends on active involvement. Anyone interested in getting involved with Open Source with SLU projects can sign up by opening a [Pull Request (PR)](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) on the [contributors](contributors.md) page in this website. The PR will be merged when the Pull Request includes a link to a merged ticket from a project in the [portfolio](../portfolio.md) that they made contribution to, and the PR has been approved by the current tech lead on that project.
+
+If you want to be recognized as a contributor, your pull request should include your name and a link to the work you have completed on an Open Source with SLU project.
+
+* [ ] Add your name to the list in `docs/about/contributors.md`;
+  * Each entry must include a name;
+  * Each entry may optionally include a link to a GitHub user profile;
+  * Add your entry to the end of the list.
+* [ ] Add a link to an already merged Pull Request or a closed Issue in an OSS [project](https://oss-slu.github.io/docs/portfolio) to your Pull Request. You can use the autolinked references formats for [issues and pull requests](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls#issues-and-pull-requests).
+
+## Join the `oss-slu` Team
+
+After making five contributions, you can be added to the `oss-slu` team in GitHub by contacting one of the program staff. Send an e-mail to <oss@slu.edu> or a slack message to the current Tech Lead on the project(s) you made contributions to. Include a list of at least five closed Issues and/or merged Pull Requests that you worked on.
+ 
 ## Git
 
-All of our projects are stored in public GitHub repositories. To make code contributions to our repositories, you will need to know the basics about Git and GitHub. We recommend this <a href="https://youtu.be/RGOj5yH7evk">video tutorial</a>, if you are not familiar with these tools.
+All our projects are stored in public GitHub repositories. To make code contributions to our repositories, you will need to know the basics about Git and GitHub. We recommend this [video tutorial]("https://youtu.be/RGOj5yH7evk"), if you are not familiar with these tools.
```

Comment:
The quotes in the link make the link not show up

Reply:
Interesting! The quotes seem to force Docusaurus to interpret it as a relative link rather than an absolute URL. All these should be fixed now.